src/select.coffee

```diff
@@ -80,8 +81,19 @@ Ember.AddeparMixins.ResizeHandlerMixin, Ember.Widgets.KeyboardHelper,
   # we need to set tabindex so that div responds to key events
   tabindex: 0
 
+  ###*
+  * Flag controlling the dropdown's appearance of the select component
+  * @type {Boolean}
+  ###
   showDropdown: no
 
+  ###*
+  * Flag indicating that dropdown will always appear as part of the select
+  * component
+  * @type {Boolean}
+  ###
+  lockDropdownOnShow: no
```

Comment:
Can we prefix this with `is` or `should`? Kind of a weird name too, shouldInitWithDropdown or something along those lines?